codespell_lib/data/dictionary.txt

```diff
@@ -18372,10 +18373,15 @@ obstacal->obstacle
 obstancles->obstacles
 obstruced->obstructed
 obsure->obscure
+obtaien->obtain
 obtaiend->obtained
+obtaiens->obtains
```

Comment:
Again

Reply:
Agreed there are a handful of examples on Google of this too, although a lot seem to be poor OCR. They do seem to be obtains in the ones I can find.